### PR TITLE
New Login Form

### PR DESCRIPTION
**Pull request by Bryant-24**

### CODE REVIEW

File: App-Login-Form/App-Login-Form/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    
+    @IBAction func passwordTextFieldChanged(_ sender: UITextField) {
+        checkField(sender)
+    }
+
+    @IBAction func actionLoginButton(_ sender: UIButton) {
+        if emailTextField.text == emailMock && passwordTextField.text == passwordMock {
+            performSegue(withIdentifier: "goToHomePage", sender: sender)
+        } else {
+            let alert = UIAlertController(title: "Error",
```
Comment (AntonPastrevich) [inline App-Login-Form/App-Login-Form/ViewController.swift:57]: What about localization?)

File: App-Login-Form/App-Login-Form/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        super.viewDidLoad()
+        
+        emailTextField.delegate = self
+        passwordTextField.delegate = self
+        setEnabledLoginButton(false)
+    }
+    
+    // MARK: - IBActions
+    @IBAction func emailTextFieldChanged(_ sender: UITextField) {
+        checkField(sender)
```
Comment (AntonPastrevich) [inline App-Login-Form/App-Login-Form/ViewController.swift:46]: I like it :)

**Review by AntonPastrevich — APPROVED** [review]
Good job

File: App-Login-Form/App-Login-Form/ViewController.swift (hunk 1 above)
Comment (Bryant-24) [reply App-Login-Form/App-Login-Form/ViewController.swift:57]: Let's with Eng only ;)

**Pull request merged**